thorlcr/activities/lookupjoin/thlookupjoinslave.cpp

```diff
@@ -2255,8 +2255,10 @@ class CLookupJoinActivityBase : public CInMemJoinBase<HTHELPER, IHThorHashJoinAr
             }
             virtual bool spill(bool critical) // called from OOM callback
             {
+                if (!channelCollector->spill(critical))
+                    return false;
```

Comment:
Separate issue - I think the use of atomic_cas in putRow() is probably overkill, unless there can be concurrent calls to putRow()